html/infrastructure/safe-passing-of-structured-data/shared-array-buffers/broadcastchannel-success.https.html

```diff
@@ -6,30 +6,27 @@
 <link rel="author" title="Domenic Denicola" href="mailto:d@domenic.me">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
+<script src="/common/utils.js"></script>
 
 <div id="log"></div>
 
 <script>
 "use strict";
 
 promise_test(t => {
+  const channelName = token();
   return Promise.all([
-    createIFrame("resources/broadcastchannel-iframe.html"),
-    createIFrame("resources/broadcastchannel-iframe.html"),
-    createIFrame("resources/broadcastchannel-iframe.html")
+    createIFrame(`resources/broadcastchannel-iframe.html?channel=${channelName}&index=0`),
+    createIFrame(`resources/broadcastchannel-iframe.html?channel=${channelName}&index=1`),
+    createIFrame(`resources/broadcastchannel-iframe.html?channel=${channelName}&index=2`)
   ]).then(() => {
     const sab = new SharedArrayBuffer(3);
     const view = new Uint8Array(sab);
-    const channel = new BroadcastChannel("channel name");
+    const channel = new BroadcastChannel(channelName);
 
     return new Promise(resolve => {
       let soFar = 0;
-      channel.onmessage = t.step_func(({ data: { sab: broadcastSAB, i } }) => {
-        if (broadcastSAB) {
```

Comment:
Why is removing this OK?

Reply:
Because the current channel is not a destination and none of the destinations transmit a sab.